SASS_spasalon/styles/_contact-us.scss

```diff
@@ -0,0 +1,174 @@
+@import "variables";
+
+.work-hours p {
+    margin: 10px 0;
+}
+
+.work-adress p {
+    width: 210px;
+    line-height: 25px;
+}
+
+nav {
+    height: 60px;
+    width: 100%;
+    margin: $main-element-margin;
+    background: $main-heder-color;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    a {
+        font-family: Acme;
+        color: $main-button-color;
+        text-decoration: none;
+        font-size: 25px;
+        padding: $main-element-padding;
+        height: 60px;
+        line-height: 60px;
+
+        &:hover {
+            background: $main-hover-color;
+        }
+        &:nth-child(2) {
+            margin: 0 120px;
+        }
+    } 
+}
+
+.contactus {
+    width: 770px;
+    margin: 150px auto 200px;
+}
+
+h1 {
+    text-align: center;
+    font-size: 2em;
+}
+
+h2 {
+    margin-bottom: 0;
+    font-size: 2em;
+    text-align: left;
+}
+
+.contact-info p {
+    margin-top: 0;
+    margin-bottom: 10px;
+    font-size: 1.5em;
+}
+
+.contact-info span {
+    margin-top: 0;
+    margin-bottom: 10px;
+    font-size: 1.5em;
+    text-decoration: underline;
+}
+
+.address-map iframe {
+    border: 1px solid $main-border-color !important;
+    filter: grayscale(0.8);
+}
+
+.form-position {
+    position: relative;
+    height: 750px;
+}
+
+.contact-form {
```

Comment:
you have duplication of this selector below
so you had to rewrite it in css style